x-pack/legacy/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -1206,157 +1206,167 @@ export class TimeSeriesExplorer extends React.Component {
           jobs.length > 0 &&
           (fullRefresh === false || loading === false) &&
           hasResults === true && (
-            <div>
+            <div className="results-container">
```

Comment:
This change now contradicts the the comment below (`... inside this **plain** wrapping element ...`) can you check this doesn't break tooltip positioning? Depending on the outcome we should fix the `div` or adapt the comment.

Reply:
Good spot. I have added the extra plain `div` back in (one with 0px left/right padding) in https://github.com/elastic/kibana/pull/56741/commits/5efc092bdd452736f99ac3d49f611cd710ff86ff. I also edited the comment used in `explorer.js` to clarify the role of the container div for the `ChartTooltip`.